validation/exclusions_test.go

```diff
@@ -37,20 +37,33 @@ var exclusions = map[string]map[uint64]bool{
 	ChainIDRPCTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+		12345:    true, // sepolia/opdeployer
 	},
 	GenesisRPCTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+		12345:    true, // sepolia/opdeployer
 	},
 	UniquenessTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   Not in https://github.com/ethereum-lists/chains
 		11763072: true, // sepolia-dev-0/base-devnet-0     Not in https://github.com/ethereum-lists/chains
+		12345:    true, // sepolia/opdeployer
```

Comment:
skipped because got this error:

```
=== FAIL: github.com/ethereum-optimism/superchain-registry/validation TestValidation/op-deploy-sepolia-test_(12345)/Uniqueness (0.00s)
    uniqueness_test.go:124: 
                Error Trace:    /Users/blaine/code/superchain-registry/validation/uniqueness_test.go:124
                                                        /Users/blaine/code/superchain-registry/validation/validation_test.go:43
                Error:          Not equal: 
                                expected: "Step Testnet"
                                actual  : "op-deploy-sepolia-test"
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -Step Testnet
                                +op-deploy-sepolia-test
                Test:           TestValidation/op-deploy-sepolia-test_(12345)/Uniqueness
                Messages:       Local chain name for 12345 does not match name from chainid.network
    uniqueness_test.go:130: 
                Error Trace:    /Users/blaine/code/superchain-registry/validation/uniqueness_test.go:130
                                                        /Users/blaine/code/superchain-registry/validation/validation_test.go:43
                Error:          []string{"https://rpc.testnet.step.network/"} does not contain "http://awe.some.rpc/"
                Test:           TestValidation/op-deploy-sepolia-test_(12345)/Uniqueness
                Messages:       Specified RPC not specified in chainid.network
```

not unique. For this it doesn't need to be.